src/azure-cli/azure/cli/command_modules/databoxedge/tests/latest/test_device_scenario_coverage.md

```diff
@@ -1,8 +1,8 @@
 |Scenario|Result|ErrorMessage|ErrorStack|ErrorNormalized|StartDt|EndDt|
-|step_device_create|successed||||2021-01-13 08:31:38.209522|2021-01-13 08:31:43.981223|
-|step_device_show|successed||||2021-01-13 08:31:56.811539|2021-01-13 08:31:59.056123|
-|step_device_list|successed||||2021-01-13 08:32:01.747769|2021-01-13 08:32:05.003301|
-|step_device_list2|successed||||2021-01-13 08:31:52.250896|2021-01-13 08:31:54.480896|
-|step_device_update|successed||||2021-01-13 08:31:54.481895|2021-01-13 08:31:56.810543|
-|step_device_delete|successed||||2021-01-13 08:31:59.068123|2021-01-13 08:32:01.747769|
+|step_device_create|successed||||2021-02-09 05:32:14.712695|2021-02-09 05:32:15.848727|
```

Comment:
just curious, why need change databoxedge file?

Reply:
This was automatically generated by adev test. Just reverted these unrelated changes.